Signal/src/ViewControllers/ColorPickerViewController.swift

```diff
@@ -66,17 +66,21 @@ class ColorView: UIView {
         self.addSubview(selectedRing)
         self.addSubview(swatchView)
 
-        let cellHeight: CGFloat = 64
-
+        // Selected Ring
+        let cellHeight: CGFloat = ScaleFromIPhone5(60)
```

Comment:
At some point we should huddle with Myles and make sure he likes the `ScaleFromIPhone5()` approach to responsive layout.